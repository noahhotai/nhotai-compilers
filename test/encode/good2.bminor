"dfsadflksdahfpnosdhfnlasdhklnvasdjhfnvlsadjhfnlsadv\n\n\n\n\n\njhfnlsa;dhfnlsadjhflaskdjnflasdkhfnlsdkjhfnlskdhflkshf"