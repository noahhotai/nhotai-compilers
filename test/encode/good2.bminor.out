Decoded line: dfsadflksdahfpnosdhfnlasdhklnvasdjhfnvlsadjhfnlsadv





jhfnlsa;dhfnlsadjhflaskdjnflasdkhfnlsdkjhfnlskdhflkshf
Encoded line: "dfsadflksdahfpnosdhfnlasdhklnvasdjhfnvlsadjhfnlsadv\n\n\n\n\n\njhfnlsa;dhfnlsadjhflaskdjnflasdkhfnlsdkjhfnlskdhflkshf"
